### DISCUSSION TITLE
copilot.nvim does not do autocomplete until running :Copilot commands

### DISCUSSION THREAD

**Discussion by plzfgme** [community/community#68530] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

In neovim, I can get autocomplete only after running some `:Copilot` commands. How to enable it by default on startup?

**Comment by Akash1134** [comment, marked as answer]
Hi @plzfgme ,

Have you tried adding the init command to your Neovim configuration file (`init.vim` or `init.lua` depending on your setup) ?


   For `init.vim`:
   ```vim
   autocmd VimEnter * Copilot setup
   ```

   For `init.lua`:
   ```lua
   vim.cmd [[autocmd VimEnter * Copilot setup]]
   ```

Thanks!
